pytest/tests/sanity/state_sync2.py

```diff
@@ -4,16 +4,22 @@
 # and sync
 
 import sys, time
+import fcntl
 
 sys.path.append('lib')
 
 from cluster import start_cluster
 from utils import LogTracker
 
+fcntl.fcntl(1, fcntl.F_SETFL, 0) # no cache when execute from nightly runner
+
+print('start state sync2')
```

Comment:
remove at least one of the two traces?

Reply:
want to keep both to see do stdout and stderr both works